sql/update/populate_customer_in_projects.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/*
+ * Copyright (C) 2016 Igalia, S.L. <info@igalia.com>
+ *
+ * This file is part of PhpReport.
+ *
+ * PhpReport is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PhpReport is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PhpReport.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * This script selects the customer id related to each project and populate the new customerid
+ * column in project table.
+ */
+
+define('DB_NAME', 'phpreport');
+define('DB_USER', 'phpreport');
+define('DB_PASSWORD', 'phpreport');
+define('DB_HOST', 'localhost');
+define('DB_PORT', 5432);
+
+
+define('PHPREPORT_ROOT', __DIR__ . '/../../');
+include_once(PHPREPORT_ROOT . '/util/DBPostgres.php');
+
+$link = pg_connect("host=".DB_HOST.
+	" port=".DB_PORT.
+	" user=".DB_USER.
+	" dbname=".DB_NAME.
+	" password=".DB_PASSWORD);
+if (!$link) error_log("ERROR: Could not connect to DB");
+
+$result = pg_query($link, $query = "SELECT customerid, projectid FROM requests");
+if ($result == NULL) error_log("ERROR: Could not run query: $query");
+$frequency = array();
+
+while ($row=pg_fetch_array($result, NULL, PGSQL_ASSOC)) {
+	$frequency[$row['projectid']][$row['customerid']] = true;
+}
+
+$projectCustomerFreq = array();
```

Comment:
You don't really need this array, as you don't need to store the values outside the foreach loop.